src/tests/unit/tests/DetailsView/details-view-body.test.tsx

```diff
@@ -104,7 +104,7 @@ describe('DetailsViewBody', () => {
                 tabStoreData: new TabStoreDataBuilder().build(),
                 visualizationStoreData: new VisualizationStoreDataBuilder().build(),
                 visualizationScanResultData: new VisualizationScanResultStoreDataBuilder().build(),
-                featureFlagStoreData: { reflowUI: true } as FeatureFlagStoreData,
```

Comment:
Do we still need the feature flag store in this test?

Reply:
Since it's in the props, I'd like to keep it here. My guess is that's how it was before the reflow changes as well.

Reply:
Same for below.